docs/feature_leader_key.md

```diff
@@ -70,3 +71,69 @@ SEQ_THREE_KEYS(KC_C, KC_C, KC_C) {
   SEND_STRING("Per key timing is great!!!");
 }
 ```
+
+## Customization 
+
+The Leader Key feature has some additional customization that you can perform here.  It has two functions that can be called at certain parts of the process.  Namely `leader_start()` and `leader_end()`.
```

Comment:
I think that the last part of the first sentence can be confusing, what does `here` means, for example ? I will short it to just `The Leader Key feature offers some additional customization`